loadgen/throughputstress/throughput_stress.go

```diff
@@ -92,40 +93,43 @@ func ParseAndValidateSleepActivityConfig(jsonStr string) (*SleepActivityConfig,
 }
 
 // Sample generates a list of SleepActivityInput instances based on the SleepActivityConfig.
-func (config *SleepActivityConfig) Sample() []*kitchensink.ExecuteActivityAction {
+func (config *SleepActivityConfig) Sample(seed int64) []*kitchensink.ExecuteActivityAction {
 	if config == nil {
 		return nil
 	}
 
-	count, ok := config.Count.Sample()
+	count, ok := config.Count.Sample(seed)
```

Comment:
using the same `seed` multiple times here is a little worrying, the selections may be correlated. using the seed once and then passing around the rand.Rand seems better.